src/Captioning/Format/SubripFile.php

```diff
@@ -159,9 +159,10 @@ private function validateOptions(array $options)
     /**
      * @param string $startTimeline
      * @param string $endTimeline
+     * @param boolean $allowEqual
      * @return boolean
      */
-    private function validateTimelines($startTimeline, $endTimeline)
+    private function validateTimelines($startTimeline, $endTimeline, $allowEqual = false)
```

Comment:
I don't think so because:
1. Method should perform operations in old way by default.
2. This method validate timelines both for one subtitle and for end of previous subtitle and start of next. From my point of view there is no sense to show subtitle and hide it at the same moment. Please, correct me in case of mistake.

Anyway I prefer add parameters to method in way when methods will perform operations in old way by default so you don't need to alter all calls in code :smile: